install_config/http_proxies.adoc

```diff
@@ -39,6 +39,19 @@ NO_PROXY=master.hostname.example.com
 ----
 ====
 
+[WARNING]
+====
+OpenShift does not accept `*` as a wildcard attached to a domain suffix. For
+example, `*NO_PROXY=*.example.com*` works, but `*NO_PROXY=*.example.com*` does
```

Comment:
@ahardin-rh You can escape some of the asterisks with a `\`, but the combinations in the NO_PROXY examples may be too tricky. Might be easier to just break them up into two verbatim code blocks instead of having them both inline.

If you do that, I'd suggest moving this to after the 2-step procedure this is currently in the middle of (it's also currently breaking the numbering) and take it out of a Warning box.